promlog/log.go

```diff
@@ -26,14 +26,17 @@ import (
 	"github.com/go-kit/log/level"
 )
 
-var (
-	// This timestamp format differs from RFC3339Nano by using .000 instead
-	// of .999999999 which changes the timestamp from 9 variable to 3 fixed
-	// decimals (.130 instead of .130987456).
-	timestampFormat = log.TimestampFormat(
-		func() time.Time { return time.Now().UTC() },
-		"2006-01-02T15:04:05.000Z07:00",
-	)
+// This timestamp format differs from RFC3339Nano by using .000 instead
+// of .999999999 which changes the timestamp from 9 variable to 3 fixed
+// decimals (.130 instead of .130987456).
+var timestampFormat = log.TimestampFormat(
+	func() time.Time { return time.Now().UTC() },
+	"2006-01-02T15:04:05.000Z07:00",
+)
+
+const (
+	LevelFlagOptions  = "debug, info, warn, error"
```

Comment:
I would go the other way, make this a slice literal and `strings.Join()` in the help function.

```suggestion
var LevelFlagOptions  = []string{"debug", "info", "warn", "error"}
```

Reply:
I thought about doing that but it seemed like a waste because all we need is a string...

Reply:
We have to do one string join/split no matter what we do. It makes more sense to join than to split.